erp-data-exchange-specification.md

```diff
@@ -304,15 +304,93 @@ These options are outlined here:
 | **reportEmails**              | Array of Strings                                  | List of email addresses which should receive report emails for this job                                                                                           |
 | **unitType**                  | string                                            | Controls the type of import Units, one of type 'rented' or 'owned'                                                                                                |
+| **webhooks**                  | Array of [webhooks](#webhooks)                    | List of [webhooks](#webhooks) that can be called for each job, triggered by defined events.                                                                       |
 
 #### Example
 
 ```json
 {
-  "autoImport": false,
-  "locale": "de_DE",
-  "receiveAdminNotifications": false,
-  "reportEmails": ["mr.foo@bar.test", "mrs.foo@bar.test"]
+	"autoImport": false,
+	"locale": "de_DE",
+	"receiveAdminNotifications": false,
+	"reportEmails": ["mr.foo@bar.test", "mrs.foo@bar.test"],
+	"webhooks": [
+		{
+			"description": "allthings#slackbot-noise",
+			"format": "slack",
+			"triggers": true,
+			"url": "https://hooks.slack.com/services/"
+		}
+	]
+}
+```
+
+### webhooks
+
+If webhooks are defined, each webhook is called when the defined event triggers. The webhooks can be defined in the import configuration or in the manifest.json.
+The webhooks from the manifest.json are merged to the configuration.  
+With this, a customer can define a webhook and also the event when it should be called.
+
+The options webhooks needed are outlined here:
+
+#### Fields
+
+| Field           | Type                        | Description                                                                                                                                        |
+| --------------- | --------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **description** | string                      | A description or name of the webhook.                                                                                                              |
+| **format**      | string                      | Either `'slack'` or `'rest'`                                                                                                                       |
+| **triggers**    | Array of strings OR boolean | Either an array of trigger-events or `true` to match all. Possible trigger events are: `'imported', 'created', 'failed', 'processing'`             |
+| **url**         | string                      | The url of the webhook which is called with a `put` request. This url can include some parameters like f.e. `jobID` which are just passed through. |
```

Comment:
Why `put` rather than `post` ?

Reply:
Thanks a lot four your review.
`put`, because the job in the adapter waits to be updated by the webhook.
So there's no new data to be created (what `post` is for mainly) and no partially update (what `patch` would be used for), but data to be updated.